src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -5063,4 +5061,112 @@ export class ResultsViewPageStore {
             return isMixedReferenceGenome(this.studies.result);
         }
     }
+
+    readonly clinicalTrackOptions = remoteData({
```

Comment:
moved from `AddClinicalTracks.tsx`

Reply:
why not put these into `AddTracks.tsx`?

Reply:
@adamabeshouse good point, I will move them to `AddTracks.tsx`